website/blog/2025-01-20-podman-quadlet.md

```diff
@@ -0,0 +1,199 @@
+
+Alternatively, navigate to the Extensions page within Podman Desktop to install it
+
+### Listing Quadlets :clipboard:
+
+On the Podman Quadlet page, you can view all the Quadlets available across your Podman machines. To update the list, click **Refresh**.
+
+<ThemedImage
+alt="Quadlets List"
+sources={{
+    light: require('./img/podman-quadlet/podman-quadlet-home-light.png').default,
+    dark: require('./img/podman-quadlet/podman-quadlet-home-dark.png').default,
+  }}
+/>
+<br/><br/>
+
+In Podman Desktop, you can see that a dedicated icon is used for the containers managed by a Quadlet.
+
+<ThemedImage
+alt="Container Quadlet Icon"
+sources={{
+    light: require('./img/podman-quadlet/container-icon-quadlet-light.png').default,
+    dark: require('./img/podman-quadlet/container-icon-quadlet-dark.png').default,
+  }}
+/>
+
+### Generate Quadlets :hammer:
+
+To generate a Quadlet from an existing container, you’ll need to install [Podlet](https://github.com/containers/podlet). The extension simplifies installation.
+
+Use one of the following ways to install Podlet:
+
+- Go to **<Icon icon="fa-solid fa-cog" size="lg" /> Settings > CLI Tools** and install Podlet using the Podman Quadlet extension.
+- Download Podlet manually from its [GitHub release page](https://github.com/containers/podlet/releases).
+
+<ThemedImage
+alt="Podlet Installation"
+sources={{
+    light: require('./img/podman-quadlet/cli-podlet-light.png').default,
+    dark: require('./img/podman-quadlet/cli-podlet-dark.png').default,
+  }}
+/>
+
+#### Example: Generate a Container Quadlet
+
+1. Start a container using Podman:
+
+```shell
+podman run --name nginx-demo -d -p 80:8080 nginx
+```
+
+2. In Podman Desktop, find your container on the Containers page.
+3. Click the **overflow menu** icon and select **Generate Quadlet**.
+
+<ThemedImage
+alt="Container actions"
+sources={{
+    light: require('./img/podman-quadlet/generate-quadlet-action-light.png').default,
+    dark: require('./img/podman-quadlet/generate-quadlet-action-dark.png').default,
+  }}
+/>
+<br/><br/>
+
+4. Click **Generate** to finalize the Quadlet.
+
+<ThemedImage
+alt="Quadlet Generate Form"
+sources={{
+    light: require('./img/podman-quadlet/generate-form-options-light.png').default,
+    dark: require('./img/podman-quadlet/generate-form-options-dark.png').default,
+  }}
+/>
+<br/><br/>
+
+5. Optional: Edit the Quadlet configuration details.
+6. Click **Load into machine**
+
+<ThemedImage
+alt="Quadlet Generate Form"
+sources={{
+    light: require('./img/podman-quadlet/generate-form-edit-light.png').default,
+    dark: require('./img/podman-quadlet/generate-form-edit-dark.png').default,
+  }}
+/>
+<br/><br/>
+
+Congrats 🎉 you created your first Quadlet!
+
+### Edit Quadlets :pen:
+
+The Quadlet details page provides three tabs:
```

Comment:
```suggestion
Click the Quadlet **STATUS** icon to view its details page, which has three tabs:
```

Reply:
It is useful to mention how to access the details page. Usually, we click the service name, but here we need to click the STATUS icon to access the page.

Reply:
This is technically a bug I opened https://github.com/podman-desktop/extension-podman-quadlet/issues/217